packages/launchpad/src/signingcosmosclient.spec.ts

```diff
@@ -27,6 +27,68 @@ describe("SigningCosmosClient", () => {
       const client = new SigningCosmosClient(httpUrl, faucet.address, wallet);
       expect(client).toBeTruthy();
     });
+
+    it("can be constructed with custom fees", async () => {
+      const wallet = await Secp256k1Wallet.fromMnemonic(faucet.mnemonic);
+      const feeOptions = {
+        feeToken: "utest",
+        gasPrice: 3.14,
+      };
+      const client = new SigningCosmosClient(httpUrl, faucet.address, wallet, feeOptions);
+      expect((client as any).fees).toEqual({
```

Comment:
Goo to test this. You can create a `PrivateSigningCosmosClient` similar to `PrivateCosmWasmClient` that allows you to access this in a type safe way.